sparkle/gui/stim/abstract_stim_editor.py

```diff
@@ -58,9 +58,10 @@ def closeEvent(self, event):
         if not msg:
             msg = self.model().warning()
         if msg:
-            answer = QtGui.QMessageBox.question(self, 'Oh Dear!', 
-                                'Problem: {}. Do you want to deal with this?'.format(msg),
-                                QtGui.QMessageBox.Yes | QtGui.QMessageBox.No)
-            if answer == QtGui.QMessageBox.Yes:
+            answer = QtGui.QMessageBox( QtGui.QMessageBox.Warning, ' ', 'Warning: {}. Do you want to change this?'.format(msg) )
```

Comment:
Give the message box a title (perhaps something more useful than mine, like "Invalid Settings")